src/test/java/it/TestApplication.java

```diff
@@ -17,6 +17,6 @@
 
 import org.junit.Test;
 
-public class TestApplication extends EndpointTest {
+public class TestApplication extends EndpointIT {
```

Comment:
Not quite sure what the point of this test is, but if you don't rename the file ApplicationIT.java, surefire will try to run during unit test and it will fail.  If you rename it ApplicationIT.java, then you end up running the same thing twice and that doesn't make sense.